src/Chapter1.hs

```diff
@@ -301,43 +301,43 @@ expressions in GHCi
   functions and operators first. Remember this from the previous task? ;)
 
 >>> 1 + 2
-<INSERT THE RESULT INSTEAD OF THE TEXT>
+3
 
 >>> 10 - 15
-<INSERT THE RESULT INSTEAD OF THE TEXT>
+(-5)
```

Comment:
Apparently, rendering of negative literals has changed in recent GHC versions :rofl: So, to make doctests pass, need to change this line as well.
```suggestion
-5
```